azure_pipelines/pr-validation.yml

```diff
@@ -83,4 +83,31 @@ jobs:
       testResultsFiles: '$(Agent.BuildDirectory)/s/build/reports/*'
       failTaskOnFailedTests: true
       testRunTitle: 'Test Run - $(target)'
-  
+
+- job: 'Validate_SPM_Integration'
+  displayName: Validate SPM Integration
+  pool:
+    vmImage: 'macOS-13'
+    timeOutInMinutes: 15
+  workspace:
+    clean: all
+
+  steps:
+
+  - checkout: self
+    clean: true
+    submodules: true
+    fetchDepth: 1
+    persistCredentials: true
+    path: s
+
+  - script: |
+      /bin/bash -c "sudo xcode-select -s /Applications/Xcode_15.0.1.app"
```

Comment:
nit: is there a reason we're using Xcode 15 while in the previous job we use Xcode 14?

Reply:
not really, I've adjusted it to Xcode 14